include/boost/fusion/container/generation/ignore.hpp

```diff
@@ -27,7 +29,7 @@ namespace boost { namespace fusion
     }
 
     //  "ignore" allows tuple positions to be ignored when using "tie".
-    BOOST_CONSTEXPR detail::swallow_assign const ignore = detail::swallow_assign();
+    BOOST_CONSTEXPR_OR_CONST detail::swallow_assign ignore = detail::swallow_assign();
 }}
```

Comment:
I can't find the documentation of BOOST_CONSTEXPR_OR_CONST. It's the first time I've seen this. Could you point me to the docs?